### PR TITLE
Basic sync with demo app

### PR DESCRIPTION
**Pull request by langsmith**
This pr updates this repo with the lastest demo app additions

https://github.com/mapbox/mapbox-android-demo/pull/1297

https://github.com/mapbox/mapbox-android-demo/pull/1290

https://github.com/mapbox/mapbox-android-demo/pull/1289

### CODE REVIEW

**Comment by langsmith** [comment]
Straight line distance example is looking good.

Screen Shot 2020-02-07 at 11 26 51 AM
Screen Shot 2020-02-07 at 11 26 57 AM

**Comment by langsmith** [comment]
Cards for other two examples are good

Screen Shot 2020-02-07 at 11 26 47 AM

Screen Shot 2020-02-07 at 11 26 07 AM

**Comment by langsmith** [comment]
Vertical videos are looking broken and waaaayyy too wide long when I test locally 😨 . I've never seen this before 🤔 cc @colleenmcginnis @katydecorah 

Screen Shot 2020-02-07 at 11 27 19 AM

Screen Shot 2020-02-07 at 11 29 04 AM

**Comment by langsmith** [comment]
Other vertical videos look to be ok. Is it something about testing locally? The videos in this pr will end up ok?

Following vertical videos are looking fine:

- https://docs.mapbox.com/android/maps/examples/recycler-view-directions/
- https://docs.mapbox.com/android/maps/examples/adjust-a-layers-opacity/
- https://docs.mapbox.com/android/maps/examples/api-response-icon-update/
- https://docs.mapbox.com/android/maps/examples/recycler-view-directions/

File: src/pages/java/examples/static-image-notification.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+- "import AndroidActivityToggle from '../../../components/context-dependent/android-activity-toggle';"
+- "import javaCode from '../../../../examples/maps/MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/javaservices/StaticImageNotificationActivity.java';"
+- "import xmlLayoutCode from '../../../../examples/maps/MapboxAndroidDemo/src/main/res/layout/activity_static_image_api_notification.xml';"
+---
+
+{{
+  <AndroidDeviceFrame
+    src={videoStaticImageNotificationActivity}
+    title="Use the Tilequery and Static Images APIs to eventually create notifications with helpful information and images."
+    rotation="vertical"
```
Comment (colleenmcginnis) [inline src/pages/java/examples/static-image-notification.md]: I think this should fix the device frame issues you're seeing @langsmith: 

```suggestion
    rotation="portrait"
```

File: src/pages/maps/examples/property-icon-determination.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+- "import AndroidActivityToggle from '../../../components/context-dependent/android-activity-toggle';"
+- "import javaCode from '../../../../examples/maps/MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/dds/PropertyIconDeterminationActivity.java';"
+- "import xmlLayoutCode from '../../../../examples/maps/MapboxAndroidDemo/src/main/res/layout/activity_dds_property_icon_switch.xml';"
+---
+
+{{
+  <AndroidDeviceFrame
+    src={videoPropertyIconDeterminationActivity}
+    title="Create a SymbolLayer and set the iconId to be dependent on each Feature's property key-value pair. "
+    rotation="vertical"
```
Comment (colleenmcginnis) [inline src/pages/maps/examples/property-icon-determination.md]:
```suggestion
    rotation="portrait"
```

File: src/pages/java/examples/static-image-notification.md (hunk 1 above)
Comment (langsmith) [reply src/pages/java/examples/static-image-notification.md]: :homer-into-hedges:

File: src/pages/maps/examples/property-icon-determination.md (hunk 2 above)
Comment (langsmith) [reply src/pages/maps/examples/property-icon-determination.md]: :homer-corgi-into-hedges:

**Comment by langsmith** [comment]
Ok @colleenmcginnis , everything's looking good


Screen Shot 2020-02-07 at 4 09 23 PM
Screen Shot 2020-02-07 at 4 09 38 PM

**Pull request merged**